### PR TITLE
Add Lifewire

### LINKED ISSUE #101: Add LifeWire article to Media section
Source: https://www.lifewire.com/the-tldr-act-could-help-you-make-sense-of-terms-of-service-agreements-5216643

Notify Justin A. when done.

### PR DESCRIPTION
**Pull request by clementbiron**
Closes #101.

### CODE REVIEW

File: src/translations/en/media.mdx [hunk 1]
```diff
@@ -1,3 +1,19 @@
+### Lifewire
+
+> However, Matti Schneider from the Open Terms Archive (OTA) project that follows changes to the ToS for more than 200 digital platforms responded to Poteat saying that the authors of the TLDR bill didn't pen the bill in isolation and did reach out to those working towards adding transparency to the ToS, including the OTA project.
+
+<a
+  target="_blank"
+  rel="noopener"
+  className="float__right"
+  href="https://www.lifewire.com/the-tldr-act-could-help-you-make-sense-of-terms-of-service-agreements-5216643"
+  title="An article to be found on the website of Lifewire"
```
Comment (MattiSG) [inline src/translations/en/media.mdx:10]:
```suggestion
  title="The TLDR Act Could Help You Make Sense of Terms of Service Agreements — Lifewire"
```

**Review by MattiSG — APPROVED** [review]

File: src/translations/fr/media.mdx [hunk 2]
```diff
@@ -1,3 +1,19 @@
+### Lifewire
+
+> However, Matti Schneider from the Open Terms Archive (OTA) project that follows changes to the ToS for more than 200 digital platforms responded to Poteat saying that the authors of the TLDR bill didn't pen the bill in isolation and did reach out to those working towards adding transparency to the ToS, including the OTA project.
+
+<a
+  target="_blank"
+  rel="noopener"
+  className="float__right"
+  href="https://www.lifewire.com/the-tldr-act-could-help-you-make-sense-of-terms-of-service-agreements-5216643"
+  title="Un article à retrouver sur le site de Lifewire"
```
Comment (MattiSG) [inline src/translations/fr/media.mdx:10]:
```suggestion
  title="The TLDR Act Could Help You Make Sense of Terms of Service Agreements — Lifewire"
```

**Comment by MattiSG** [comment]
Could this be deployed please? 🙂

**Comment by clementbiron** [comment]
Done.

**Pull request merged**